src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import Header from './Components/Header/Header';
+import MissionPage from './Pages/Missions/Missions';
+import Rockets from './Pages/Rockets/Rockets';
+import { getRockets } from './Redux/Rockets/rockets';
+import Profile from './Pages/Profile/Profile';
+
+const App = () => {
+  const dispatch = useDispatch();
+  dispatch(getRockets());
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20you%20only%20dispatch%20those%20actions%20once%20and%20do%20not%20add%20data%20to%20store%20on%20every%20re%2Drender%20(i.e.%20when%20changing%20views%20/%20using%20navigation).), Please make sure that actions for getting rockets dispatched once because when I load the rockets page the actions dispatched twice.

![image](https://user-images.githubusercontent.com/68796232/208939970-fb9b49ab-4d58-4af0-ab53-834eb19efafd.png)